rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.0-release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= RPA Bot 1.6.0 Release Notes
+
+RPA Bot is a software robot that runs in an on-premise environment and executes the assigned automations.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== February 21, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* Added a `Bot Service State` entry to the System Settings panel.
```

Comment:
Suggest adding the format of What's New to for all entries like in the in General Guidelines section of What's New Guidelines to something like:
* The System Settings panel now has a `Bot Service State` entry.
Maybe this guideline does not apply for RPA Bots RN